pkg/converter/convert_unix.go

```diff
@@ -140,6 +140,11 @@ func unpackNydusBlob(bootDst, blobDst string, ra content.ReaderAt, unpackBlob bo
 		defer blob.Close()
 
 		if _, err = UnpackEntry(ra, EntryBlob, blob); err != nil {
+			if errors.Is(err, ErrNotFound) {
+				// The nydus layer may contain only bootstrap and no blob
```

Comment:
Looks like we have to delete `blob` if noi unpack is needed

Reply:
No need for this, the converter will clean up all files in workdir.